src/commands/kv/delete_bulk.rs

```diff
@@ -14,32 +13,23 @@ use crate::terminal::message;
 
 const MAX_PAIRS: usize = 10000;
 
-pub fn delete_bulk(namespace_id: &str, filename: &Path) -> Result<(), failure::Error> {
-    let client = super::api_client()?;
-    let account_id = super::account_id()?;
-
-    // If the provided argument for delete_bulk is a json file, parse it
-    // and delete its listed keys. If the argument is a directory, delete key-value
-    // pairs where keys are the relative pathnames of files in the directory.
-    let mut data;
+pub fn delete_json(namespace_id: &str, filename: &Path) -> Result<(), failure::Error> {
     let keys: Result<Vec<String>, failure::Error> = match metadata(filename) {
         Ok(ref file_type) if file_type.is_file() => {
-            data = fs::read_to_string(filename)?;
+            let data = fs::read_to_string(filename)?;
             Ok(serde_json::from_str(&data)?)
         }
-        Ok(ref file_type) if file_type.is_dir() => parse_directory(filename),
-        Ok(_file_type) => {
-            // any other file types (namely, symlinks)
-            bail!(
-                "{} should be a file or directory, but is a symlink",
-                filename.display()
-            )
-        }
+        Ok(_) => bail!("{} should be a JSON file, but is not", filename.display()),
```

Comment:
we may want to smooth this and make it tell a user not only that it should be json but what the schema is; however this is mergable.

Reply:
Nice catch :)

Reply:
This might be better to put in the docs, I think? cuz the format for this can get quite long...
```
[{"key": "<KEY>", "value": "<VALUE>", "expiration": "<OPTIONAL_SECONDS>", "expiration_ttl": "<OPTIONAL_SECONDS>", "base64": "<OPTIONAL_BOOL>"}]
```